code/go/0chain.net/sharder/blockstore/interface.go

```diff
@@ -0,0 +1,45 @@
+package blockstore
+
+import "github.com/go-redis/redis"
+
+var (
+	bwrecDB blockStorage
+)
+
+type (
+	blockStorage interface {
+		Delete(key string) error
+		DeleteFromHash(tablaBName, key string) error
+		DeleteSorted(key, value string) error
+		Exec() error
+		Get(key string) ([]byte, error)
+		GetCountSorted(tablaBName string) (int64, error)
+		GetFromHash(key, value string) ([]interface{}, error)
+		GetSortedRange(key string, start, stop int64) ([]string, error)
+		GetSortedRangeByScore(key string, lastBlock, count int64) []*UnmovedBlockRecord
+		Set(key string, value []byte) error
+		SetSorted(key string, score float64, value string) error
+		SetToHash(tablaBName, key, value string) error
+		StartTx() error
+	}
+)
+
+func newBlockStore(deleteExistingDB bool) {
+	bwrecDB = newBwrDB(deleteExistingDB)
+
+}
+
+func newBwrDB(deleteExistingDB bool) blockStorage {
+	db := blockStore{}
+	db.Client = redis.NewClient(&redis.Options{
+		Addr:     "localhost:6379",
```

Comment:
as @moldis pointer this use of `localhost:6379` it should not be hardcoded. It should be taken from config file. You can have input in `newBwrDB(addr , password string, db int, deleteExistingDB bool)` function